src/collections/resources/faq/meshery-faq.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Meshery FAQs"
+thumbnail: ../../../assets/images/meshery/full-logo/meshery-logo-tag-dark-text-side.png
+category: FAQ
+tags: 
+  - Meshery
+  - FAQ
+type: FAQ
+product: FAQ
```

Comment:
```suggestion
product: Meshery
```

Reply:
@asubedy just this product can be "Meshery". Rest looks good 🚀

Reply:
Oh ok. 👍